source/upgrade/important-upgrade-notes.rst

```diff
@@ -15,12 +15,21 @@ Important Upgrade Notes
 
 .. important::
    - Support for Mattermost Server v5.31 `Extended Support Release <https://docs.mattermost.com/upgrade/extended-support-release.html>`_ has come to the end of its life cycle as of October 15, 2021. Upgrading to Mattermost Server v5.37 `Extended Support Release <https://docs.mattermost.com/upgrade/extended-support-release.html>`_ or later is required.
-   - The "version" CLI command will be deprecated in Mattermost v6.1.
+   - The "version" CLI command will be deprecated in a future release.
```

Comment:
@wiersgallak I changed this from v6.1 to "a future release" because this wasn't done for v6.1 (as far as I know). https://mattermost.atlassian.net/browse/MM-37987

Reply:
Thanks - that's perfect for now.